### ISSUE TITLE
Staticman reply target error

### ISSUE THREAD

**Issue by VincentTam** [onweru/hugo-swift-theme#48]
The reply target name is incorrect.  In this JS function, you're capturing reply target name by copying the text in the `comment_name` class element.  That's incorrect in case of comment reply, which contains
```html
<span> ↷ </span><a href='#{{ .replyID }}' class='reply_target'>{{ .replyName }}</a>
```

https://github.com/onweru/hugo-swift-theme/blob/a277912732091c665b9c10458ec3c6b33ec9d580/assets/js/index.js#L355-L368
:heavy_check_mark: Main comment:
https://github.com/onweru/hugo-swift-theme/blob/cc386c77ba4f884c882d04360968253e9d77ba46/layouts/partials/comments.html#L12-L20
:x: Comment reply:
https://github.com/onweru/hugo-swift-theme/blob/cc386c77ba4f884c882d04360968253e9d77ba46/layouts/partials/comments.html#L25-L34

To accurately capture the reply target name, I've created an additional wrap for main comments in my PR for nested comments.
https://github.com/onweru/hugo-swift-theme/blob/d9567f76ae02adb5a7f798e6dd5e27470b13767f/layouts/partials/comments.html#L16
I'm going to undo your removal of the inner wrap https://github.com/onweru/hugo-swift-theme/commit/cc386c77ba4f884c882d04360968253e9d77ba46#diff-4bc063bcdcf2e635a36bd0d8d6273ad8 so as to give accurate results.

**Comment by VincentTam** [comment]
The reply target name is wrong.  In my original code, the `<div>` element for comment author contains an additional `<span>` element for wrapping the reply target.  Deleting that wouild case incorrect reply target name in case of comment reply.

**Comment by onweru** [comment]
> In this JS function, you're capturing reply target name by copying the text in the comment_name class element. That's incorrect in case of comment reply, which contains

Is the comment nesting recursive?

**Comment by VincentTam** [comment]
> Is the comment nesting recursive?

Since a *n*-level deep nested comments would be neither easy to code, practical nor beautiful, I'm only making this two-level deep, meaning

> cmt 1
> &nbsp;&nbsp; owner rpl1.1
> cmt 2
> &nbsp;&nbsp; owner rpl2.1

In this way, everyone can focus on the interested question/thread.

> I suppose I overlooked that someone can respond to a reply. Correct?

This feature is still young, not it's normal that one doesn't understand fully how this works.

Anyways, please refer my original blog article for reference:
https://vincenttam.gitlab.io/post/2018-11-17-nested-comments-in-beautiful-hugo/

I'm working to fix this on [`nestedCmt2`](https://github.com/VincentTam/hugo-swift-theme/tree/nestedCmt2) branch.

* * *

:red_circle: Please follow [Google's HTML/CSS Style Guide](https://google.github.io/styleguide/htmlcssguide.html) while coding, especially

1. Use two white spaces for indentation
2. Add a space before left curly braces, like `function name(args) {` and `.myclass {`
3. Avoid trailing spaces.  If you're using VS code, please implement settings described in https://stackoverflow.com/questions/30884131/remove-trailing-spaces-automatically-or-with-a-shortcut.  It's distracting to see sth like VincentTam/hugo-swift-theme@a154334a.
3. Avoid tag names in CSS as much as possible in favor of the use of class names.

Git commit pratices:

1. Please use [informative commit title](https://chris.beams.io/posts/git-commit/), so that we know what you're doing.
5. Please [keep Git history down](https://discourse.gohugo.io/t/hugo-themes-donts/17714?u=vincenttam).
7. Please do [only one thing in each commit](https://victoria.dev/verbose/git-commit-practices-your-future-self-will-thank-you-for/), so that your [Git commit history tells a story](https://about.futurelearn.com/blog/telling-stories-with-your-git-history).
6. Please create a PR for big changes/new features/bug fixes.  This helps the testing/debug.
6. Please run local tests (or even better, post sample screenshots) before actually merging stuff to the master branch, in order to avoid committing errors.
8. Please add `--ws-error-highlight=all` to spot out any trailing spaces.

Suggestion: try [oh-my-zsh](https://ohmyz.sh/) for Git shortcuts like

```sh
alias
g=git
ga='git add'
gaa='git add --all'
gap='git apply'
gb='git branch'
gbD='git branch -D'
gba='git branch -a'
gbd='git branch -d'
gcp='git cherry-pick'
gcam='git commit -a -m'
gf='git fetch'
gp='git push'
gsh='git show'
```

Linguistic considerations:

1. Use one translation string for one whole expression, due the difference of word order in different languages.
2. Replace strings with Unicode icons if possible to minimise the need for creating new locale strings.
3. Favor existing locale strings instead of writing your own.
4. Leave untranslated (English) string for foreign languages with some unknown translations, like https://vincenttam.frama.io/fish-demo/pl/blog/goisforlovers/ (:poland:).

**Comment by VincentTam** [comment]
Closed via #49.
